static/js/settings.js

```diff
@@ -378,7 +378,7 @@ var SettingsScreen = {
           addon.installed = this.installedAddons.has(addon.name);
           new DiscoveredAddon(addon);
         }
-      });
+      }).catch((e) => console.error(`Failed to parse add-ons list: ${e}`));
```

Comment:
Hmm. This will log to the web console right? I was thinking that we should have something log to the node console. Or is this a shared file?

Since the URL can't change at runtime, maybe we can do a validity check on the server side at startup time?

Reply:
Yes, it will log to the web console. As far as I know, we don't currently have an easy was to send errors back to node for logging. Maybe we should think about doing that.

I don't think checking at startup will do much good. The addons list can change at any time, and the gateway can be running for an indefinite period of time, so a one-off check doesn't really help.

Reply:
ok - then this is as good as we can do for now.

Maybe an even better solution is to do a travis job that runs jsonlint on each PR